src/algorithms/calorimetry/CalorimeterPulseGeneration.cc

```diff
@@ -0,0 +1,154 @@
+// SPDX-License-Identifier: LGPL-3.0-or-later
+// Copyright (C) 2025 Minho Kim
```

Comment:
```suggestion
// Copyright (C) 2024 - 2025 Simon Gardner, Chun Yuen Tsang, Prithwish Tribedy, Minho Kim
```
Let's credit original authors of the work you derive from.